contrib/go-redis/redis.v8/redis.go

```diff
@@ -134,16 +138,21 @@ func (ddh *datadogHook) BeforeProcessPipeline(ctx context.Context, cmds []redis.
 	raw := commandsToString(cmds)
 	length := strings.Count(raw, " ")
 	p := ddh.params
-	opts := make([]ddtrace.StartSpanOption, 0, 7+len(ddh.additionalTags)+1) // 7 options below + for additionalTags + for analyticsRate
+	var opts []ddtrace.StartSpanOption
 	opts = append(opts,
 		tracer.SpanType(ext.SpanTypeRedis),
 		tracer.ServiceName(p.config.serviceName),
 		tracer.ResourceName(raw[:strings.IndexByte(raw, ' ')]),
-		tracer.Tag("redis.raw_command", raw),
 		tracer.Tag("redis.args_length", strconv.Itoa(length)),
-		tracer.Tag(ext.ResourceName, raw),
 		tracer.Tag("redis.pipeline_length", strconv.Itoa(len(cmds))),
 	)
+
+	if !p.config.excludeRawCommand {
+		opts = append(opts,
+			tracer.Tag("redis.raw_command", raw),
+			tracer.Tag(ext.ResourceName, raw))
```

Comment:
Sorry for missing this. The resource name is 100% mandatory for all spans. We can not leave it out.